src/encoded/audit/experiment.py

```diff
@@ -1912,6 +2009,18 @@ def check_wgbs_lambda(bismark_metrics, threshold, pipeline_title):
                                    level='WARNING')
 
 
+def check_wgbs_lambda_ENCODE4(gembs_metrics, threshold, pipeline_title):
+    for metric in gembs_metrics:
+        conversion_rate = metric.get('conversion_rate')
+        if conversion_rate > threshold:
```

Comment:
Think this should be reversed to check values under the 99% threshold:
```suggestion
        if conversion_rate < threshold:
```